cpp/ps0001/cpp_ps0001.cpp

```diff
@@ -1,26 +1,17 @@
-/* help your brother solve summation problems
-    155
-    940
-    +168
-    --------
-    1263
-*/
-
-
 #include <iostream>
 
 int main() {
-    int num1 , num2 , num3 , result;
-
+    // prompting user for 3 numbers to add 1/2
+    short num1 , num2 , num3 , result;
     std::cout << "Enter number 1" << std::endl;
     std::cin >> num1;
     std::cout << "Enter number 2" << std::endl;
     std::cin >> num2;
     std::cout << "Enter number 3" << std::endl;
     std::cin >> num3;
 
+    // calculating & outputting the result 3/3
     result = num1 + num2 + num3;
-
-    // result---
     std::cout << "sub result is : " << result << std::endl;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as program is user-friendly ⭐